.travis.yml

```diff
@@ -22,6 +22,11 @@ addons:
       - cmake
       - cmake-data
 
+env:
+  global:
+  - TRAVIS_CI=1
```

Comment:
The variable has already been used: https://github.com/OPM/opm-output/pull/194 - but yes you are right; a better solution would have been to just use one the variables already set by Travis.